Release candidate: TR1X 4.12.2 / TR2X 1.2.1

rr- opened the pull request:
#### Checklist

- [x] I have read the [coding conventions](https://github.com/LostArtefacts/TRX/blob/master/CONTRIBUTING.md#coding-conventions)
- [x] I have added a changelog entry about what my pull request accomplishes, or it is an internal change
- [x] I have added a readme entry about my new feature or OG bug fix, or it is a different change

#### Description

This accumulates the recent bugfixes to be released as a hotfix. Since this was a bit more involved release than usual, I'd appreciate some light checks around the secrets – including the TR2 issue.

lahm86 committed:
tr1/data: fix gold braid texturing

This fixes an incorrect palette reference and leftover textured
triangles in Lara's golden braid.

Resolves #3257.

rr- committed:
stats: fix non-0 indexed secrets in TR1

Resolves #3262.

rr- committed:
tr1/cutscene: fix audio drift

Resolves #3263.

rr- committed:
tr1/inv-ring: fix depth issues w/ fade effects off

Resolves #3267.

walkawayy committed:
movable_block: fix stuck blocks if game is saved while they fall

Resolves #3274.

lahm86 committed:
tr2/data: invalidate OG secret triggers

This invalidates OG level secret triggers as they should only ever
use the pickup mechanism.

Resolves #3280.

rr- committed:
audio: revert the audio fix

Resolves #3251. Reverts 9a9e2c8463ebd49a4c257666c56c47dafa80385e.

aredfan commented:
@rr- I found an irregularity with TR2 secrets not getting counted only when Lara picks them up, as opposed to using `/secret give x`, this bug is also present in develop.  The secrets does register after I make a SG then load it.

1. `/play 16`
2. `/tp sec` and pick up the secret, repeat this step another 2 times
3. use `/secret` and observe that 0 of 3 secrets are collected
4. skip to the final stats and observe that 0 secrets are collected

rr- committed:
ui/text: fix PS1 pad missing an icon

Resolves #3288.

lahm86 commented:
This should fix the problem above. Should we do a separate commit to cherry-pick here, or how best to handle?

```diff
diff --git a/src/tr2/game/stats.c b/src/tr2/game/stats.c
index e6f50243f..490ef9277 100644
--- a/src/tr2/game/stats.c
+++ b/src/tr2/game/stats.c
@@ -132,6 +132,7 @@ void Stats_MarkSecretCollected(const GAME_OBJECT_ID obj_id)
 {
     RESUME_INFO *const resume = Savegame_GetCurrentInfo(Game_GetCurrentLevel());
     M_SetSecretFlag(&resume->stats.secret_flags, obj_id);
+    Stats_UpdateSecrets(&resume->stats);
 }
 
 bool Stats_CheckAllLevelSecretsCollected(void)
 ```

rr- committed:
ui/controls: fix reset/unbind inputs not working on controllers

Resolves #3290.

rr- commented:
> This should fix the problem above. Should we do a separate commit to cherry-pick here, or how best to handle?
> 
> ```diff
> diff --git a/src/tr2/game/stats.c b/src/tr2/game/stats.c
> index e6f50243f..490ef9277 100644
> --- a/src/tr2/game/stats.c
> +++ b/src/tr2/game/stats.c
> @@ -132,6 +132,7 @@ void Stats_MarkSecretCollected(const GAME_OBJECT_ID obj_id)
>  {
>      RESUME_INFO *const resume = Savegame_GetCurrentInfo(Game_GetCurrentLevel());
>      M_SetSecretFlag(&resume->stats.secret_flags, obj_id);
> +    Stats_UpdateSecrets(&resume->stats);
>  }
>  
>  bool Stats_CheckAllLevelSecretsCollected(void)
> ```

Yeah I'll push this directly to develop and cherry-pick here.

rr- committed:
tr2/stats: fix counting secrets

rr- commented:
Done. I'd like to include #3291 as well.

lahm86 reviewed (approved):
LGTM

rr- committed:
screenshot: fix screenshot stripping accents

Resolves #3238.

rr- committed:
inv-ring: fix RNG_DONE triggered multiple times under frame drops

Resolves #3295.

lahm86 committed:
ui/text: use dotless i for accent combinations

This uses the dotless i for combining with grave, acute, circumflex
and umlaut accents.

Resolves #3298.

Kappa971 commented:
Why were the reviewed strings not added?

rr- commented:
It's a hotfix release (as indicated by a bump only in the minor version). It conflicted because of new features that are not to be included in such releases.
For similar reasons, the config tool is still around in that release, even though it's no longer supported in the main branch.
